### PR TITLE
Updates cypress command to accept url input

### LINKED ISSUE #188: Extend .docksal/commands/cypress to allow for running cypress on other urls
`CYPRESS_BASE_URL="http://${VIRTUAL_HOST}"`

Should be the default, but like the refresh command, we should be able to add `-e` to set another domain.

Usage:

`fin cypress run -e https://example.pantheonsite.io script-name`

### PR DESCRIPTION
**Pull request by thejimbirch**
closes #188

### CODE REVIEW

File: .docksal/commands/cypress [hunk 1]
```diff
@@ -14,4 +15,8 @@ export NVM_DIR="$([ -z "${XDG_CONFIG_HOME-}" ] && printf %s "${HOME}/.nvm" || pr
 
 cd ${PROJECT_ROOT}/tests/cypress
 nvm use
-CYPRESS_BASE_URL="http://${VIRTUAL_HOST}" ./node_modules/.bin/cypress  "$@"
+
+ENVIRONMENT=${2:-"$VIRTUAL_HOST"}
```
Comment (paulsheldrake) [inline .docksal/commands/cypress:19]: I think this needs to be

`ENVIRONMENT=${2:-"https://${VIRTUAL_HOST}"}`

File: .docksal/commands/open [hunk 2]
```diff
@@ -4,4 +4,4 @@
 ##
 ## Usage: fin open
 
-open https://${VIRTUAL_HOST}${NC}
+open //${VIRTUAL_HOST}${NC}
```
Comment (paulsheldrake) [inline .docksal/commands/open:7]: Can we not keep it https always?   there are lots of external scripts and other drupal pieces that just assume we're on a secure site these days

**Review by paulsheldrake — CHANGES_REQUESTED** [review]

File: .docksal/commands/open (hunk 2 above)
Comment (thejimbirch) [reply .docksal/commands/open:7]: Updated.

File: .docksal/commands/cypress (hunk 1 above)
Comment (thejimbirch) [reply .docksal/commands/cypress]: Updated.

**Review by paulsheldrake — APPROVED** [review]

**Pull request merged**